src/app/public/modules/repeater/repeater-item.component.html

```diff
@@ -30,6 +30,25 @@
   #skyRepeaterItemLeft
 >
   <div class="sky-repeater-item-left">
+    <ng-container *ngIf="draggable">
+      <span
+        aria-live="assertive"
+        class="sky-repeater-item-reorder-assistive-text"
+        [id]="contentId + '-reorder-state'"
+      >
+        {{reorderState}}
+      </span>
+      <button
+        class="sky-btn sky-tile-tools-control sky-repeater-item-grab-handle"
+        [attr.aria-label]="reorderButtonLabel"
+        (blur)="handleBlurEvent($event)"
+        (click)="$event.stopPropagation()"
+        (keydown)="handleKeyboardEvent($event)"
+        #grabHandle
+      >
+        <sky-icon icon="arrows-v"></sky-icon>
```

Comment:
`icon` attribute should be on new line.

Reply:
Fixed quite a few of these that were throughout this file.